packages/storefront-lwc/src/modules/commerce/errorPage/errorPage.js

```diff
@@ -0,0 +1,36 @@
+/*
+    Copyright (c) 2020, salesforce.com, inc.
+    All rights reserved.
+    SPDX-License-Identifier: BSD-3-Clause
+    For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+*/
+
+import { LightningElement, api } from 'lwc';
+
+export default class ErrorPage extends LightningElement {
+    @api pagenotfound = false;
+    @api servererror = false;
```

Comment:
Is this meant to be @api?

Reply:
Yes .. We (can) pass these from where the errorPage component is included. e.g. storeFront.html